### ISSUE TITLE
Backup is still present even after we delete the HA configuration from the standby node 

### ISSUE THREAD

**Issue by Arpit-yb** [yugabyte/yugabyte-db#8974]
**Steps to repro**

1. Create two VM instances in AWS/GCP and install the 2.6 portal there
2. Create a universe in any one of the instance 
3. Create a HA configuration with that instance being the active one and add the other instance as a standby node
4. After  at least 1 replication interval is over , promote that standby node to an active one
5. Now delete the HA configuration from both the instances
6. Now , the Universe that was created is visible in both the instances , implying that we have two universes but we created only one.

**Comment by VijiYB** [comment]
@daniel-yb - Daniel can you please check if this is expected?

**Comment by daniel-yb** [comment]
@VijiYB yes that's a known thing that is already documented right now (see the attached screenshot describing this exact scenario in our docs and what we recommend the customer to do in this situation). There are some ways we can enhance the code to prevent this in the future, but there aren't any imminent plans to work on it given my focus on the cloud GA project right now.

<img width="1087" alt="Screen Shot 2021-06-18 at 12 54 52 AM" src="https://user-images.githubusercontent.com/54864769/122527041-d0037700-cfcf-11eb-9fc0-5971e481c84d.png">
